css-paint-api/Overview.bs

```diff
@@ -138,13 +138,22 @@ interface PaintWorkletGlobalScope : WorkletGlobalScope {
 };
 </pre>
 
+The {{ContextAttributes}} allows the settings for the rendering context associated
+with the paint canvas. Right now, the {{ContextAttributes}} has a boolean flag
+indicating if the paint canvas contains an alpha channel.
+<pre class='idl'>
+dictionary ContextAttributes {
```

Comment:
might be nice to namespace this for the spec e.g. {{PaintContextAttributes}}, but doesn't really matter as not user visible.

Reply:
It makes sense that we might eventually want to support settings like color space. However, in the event that we do this I think we might still only want to support a subset of CanvasRenderingContext2DSettings just like PaintRenderingContext2D implements a subset of CanvasRenderingContext2D. How about PaintRenderingContext2DSettings for consistency?

Reply:
Yes, I'd support that. PaintRenderingContext2DSettings is a better name, because it is more clear that it is used for PaintRenderingContext2D.
Question: do we need to start a discussion somewhere and bring other people onboard, like CSS discussion group?